api/v1alpha1/fenceagentsremediation_webhook.go

```diff
@@ -67,6 +75,10 @@ func (far *FenceAgentsRemediation) ValidateDelete() (admission.Warnings, error)
 	return nil, nil
 }
 
+func InitOutOfServiceTaintSupportedFlag(outOfServiceTaintSupported bool) {
```

Comment:
is there a reason to inject the flag, and not get it from the utils package like in SNR (https://github.com/medik8s/self-node-remediation/blob/5df6e10931eabaa2e4870f2bcd540fac6bbc0336/api/v1alpha1/selfnoderemediationtemplate_webhook.go#L65)? I think SNR's way is cleaner, and being consistent between both operators make maintenance easier :)

Reply:
I may misunderstand your point in https://github.com/medik8s/fence-agents-remediation/pull/92#discussion_r1462871861. So please correct me if my understanding is wrong.

Now there are two ways to enable the validation of k8s.
One is the utils package way in SNR based on a global var you mentioned.
Another is the Node Maintenance Operator way mentioned in https://github.com/medik8s/node-maintenance-operator/pull/110#discussion_r1458680419.

The following patch implemented a kind of the utils package way in SNR.
https://github.com/medik8s/fence-agents-remediation/pull/92/commits/f0f74b21f67aa49ff1b1eecc5e9e63241780e371

And the following patch implemented the Node Maintenance Operator way.
https://github.com/medik8s/fence-agents-remediation/pull/92/commits/fbc4a9714d568b42c5d49b2ea535dc05d43d36fd

How do you think which way is a better way to implement the validation?
I prefer the SNR's way because it's more simple than the Node Maintenance Operator way.
But I could not avoid using a globael variable.

Reply:
Thank you for the clarification.
I will keep the current way to avoid using a global var. But let me know if you have any comment.